### PR TITLE
TGUI Voting panel

### PR DESCRIPTION
**Pull request by TheLordME**
## Changelog

:cl:
admin: Nicer voting panel
/:cl:

### CODE REVIEW

**Comment by TheLordME** [comment]
One and a half things I still wanna fix: 
The vote reason/question not being displayed on the voting window
(opt) adding a vote action button

**Comment by TheLordME** [comment]
Question has been added

File: code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -8,7 +8,7 @@ SUBSYSTEM_DEF(ticker)
 	var/static/current_state = GAME_STATE_INIT
 
 	/// Did we attempt an automatic gamemode vote?
-	var/static/auto_gamemode_vote_attempted = FALSE
+	/// fuck this var/static/auto_gamemode_vote_attempted = FALSE
```
Comment (silicons) [inline code/controllers/subsystem/ticker.dm:11]: ?

Comment (TheLordME) [reply code/controllers/subsystem/ticker.dm:11]: I am removing the gamemode vote entirely, and on local I always get the gamemode vote, which is why I commented out (I didnt know I would get rid of the gamemode vote all together)

**Review by silicons — APPROVED** [review]

**Review by silicons — APPROVED** [review]

**Pull request merged**